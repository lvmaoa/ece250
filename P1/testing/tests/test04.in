m 2
push_front "google" "www.google.com"
print
push_front "bored" "www.boredpanda.com"
print
push_front "uw" "www.uwaterloo.ca"
print
exit

