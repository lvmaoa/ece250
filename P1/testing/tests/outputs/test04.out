success
success
"google" "www.google.com"
success
"google" "www.google.com"
"bored" "www.boredpanda.com"
success
"bored" "www.boredpanda.com"
"uw" "www.uwaterloo.ca"
